baiji/config.py

```diff
@@ -76,6 +94,13 @@ def key(self):
     def secret(self):
         return self._try(['AWS_SECRET_ACCESS_KEY', 'AWS_SECRET'], 'AWS_SECRET')
 
+    @property
+    def region(self):
+        return self._try(['AWS_DEFAULT_REGION'], 'REGION')
+
+# Alias to avoid breaking changes
+Credentials = Settings
```

Comment:
They shared the same config file parsing and resolving path. Before I thought we can extract the config resolution logic and provide both `Credentials` and `Settings` classes. Then this could just be a minor version bump, right?